resources/assets/less/landing.less

```diff
@@ -1,46 +1,70 @@
-@aboveMobile: ~"only screen and (min-width: 720px)";
-@aboveTablet: ~"only screen and (min-width: 1100px)";
+@import "./tailwind";
 
-@highlight-background-color: #648ec6;
-@secondary-background-color: #e4e7ec;
+html {
+    @apply .font-sans;
+}
 
-@nav-margin:        50px; // Margin around nav area
-@content-margin:    20%;  // Horizontal margin around content
-@section-spacing:   50px; // Vertical padding around sections
-@font-size:         20px; // 1em font size
+body {
+    @apply .mx-auto;
+    max-width: 1440px;
+}
 
-@nav-links-spacing: 15px; // Horizontal spacing between nav links
-@intro-headroom:    50px; // Spacing between nav and intro content
-@intro-gutter:      50px; // Spacing between intro screenshot and claim
-@claim-spacing:     15px; // Spacing between claim items
-@features-spacing:  20px; // Vertical spacing between features
-@footer-spacing:    20px; // Padding around footer items
+h2 {
+    @apply .font-hairline .text-double;
+}
 
-* {
-    margin: 0;
-    padding: 0;
+.btn {
+    @apply .border-0 .my-3 .text-base .py-2 .px-8 .rounded .no-underline .table .leading-large;
 }
 
-body {
-    font-family: 'Roboto', sans-serif;
-    font-size: @font-size;
-    margin: 0;
+.btn-cta {
+    @apply .bg-blue .text-white .text-lg;
 }
 
-.cta-button {
-    background: black;
-    border-radius: 5px;
-    color: white;
+.blue-overlay:before {
+    position: absolute;
+    content: " ";
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
     display: block;
-    padding: 10px;
-    text-align: center;
-    text-decoration: none;
-    text-transform: uppercase;
-}
-
-@import "landing/nav";
-@import "landing/intro";
-@import "landing/instructions";
-@import "landing/features";
-@import "landing/bottom-cta";
-@import "landing/footer";
+    z-index: 0;
+    background-color: rgba(0, 59, 87, 0.8);
+}
+
+.container {
+    @apply .pl-5 .pr-5;
+    max-width: 990px;
+}
+
+.h1-title {
+    @apply .text-5xl .font-hairline .mt-1 .text-blue-darkest;
+    font-size: 47px;
```

Comment:
Usually we try to have our fonts in rems or ems.

Also, feels like this could use a clean-up in the class name. Is this `page-title`, perhaps?